packages/usa-button-group/src/usa-button-group.twig

```diff
@@ -1,7 +1,47 @@
-<ul class="usa-button-group {{ modifier }}">
-  {% for item in items %}
+<!-- start segmented styles -->
+{% if segmented == true %}
+  <ul class="usa-button-group usa-button-group--segmented">
     <li class="usa-button-group__item">
-        {{ item }}
+      <button class="usa-button" {% if disabled_state == "disabled" %}disabled{% endif %} {% if disabled_state == "aria-disabled" %}aria-disabled="true"{% endif %}>Map</button>
```

Comment:
Maybe an if/elseif statement might be more readable  —  along with some formatting.

```suggestion
      <button class="usa-button"
        {% if disabled_state == "disabled" %}
          disabled
        {% elseif disabled_state == "aria-disabled"  %}
          aria-disabled="true"
        {% endif %}
      >
        Map
      </button>
```

Reply:
Opened issue #4881 to resolve in a future enhancement